src/signatureUtils.js

```diff
@@ -15,6 +15,7 @@ const crypto = require('crypto')
 const loggerNamespace = '@adobe/aio-lib-events'
 const logger = require('@adobe/aio-lib-core-logging')(loggerNamespace,
   { level: process.env.LOG_LEVEL })
+const ADOBE_IOEVENTS_DOMAIN = 'https://adobeioevents.com'
```

Comment:
Let's rename this to `ADOBE_IOEVENTS_SECURITY_DOMAIN` to avoid any confusion for our developers, because we also have the following domains defined:
https://github.com/adobe/aio-lib-events/blob/master/src/index.js#L31-L32

Reply:
done